### PR TITLE
Break SI Quad excdata into distinct families

### PR DESCRIPTION
**Pull request by xresende**
qdp1
qda
qdb1
q2
q4
qdp2
qfa
q1
q3
qdb2
qfb
qfp

### CODE REVIEW

File: pwrsupply/pstypes-setpoint-limits.txt [hunk 1]
```diff
@@ -86,15 +86,23 @@ ts-corrector-cv-strong-hlim  -0012.0   -0012.0   -0012.0   -0012.0   +0012.0   +
...
+si-quadrupole-q14-qdp1-fam   +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q20-q2-fam     +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q20-q4-fam     +0000.0   +0000.0   +0000.0   +0000.0   +0150.0   +0150.0   +0150.0   +0150.0   0.1       0.20      +0030.0   +0000.1
+si-quadrupole-q20-qfa-fam    +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q20-q1-fam     +0000.0   +0000.0   +0000.0   +0000.0   +0120.0   +0120.0   +0120.0   +0120.0   0.1       0.20      +0024.0   +0000.1
+si-quadrupole-q20-q3-fam     +0000.0   +0000.0   +0000.0   +0000.0   +0120.0   +0120.0   +0120.0   +0120.0   0.1       0.20      +0024.0   +0000.1
+si-quadrupole-q30-fbb-fam    +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
+si-quadrupole-q30-fbp-fam    +0000.0   +0000.0   +0000.0   +0000.0   +0160.0   +0160.0   +0160.0   +0160.0   0.1       0.20      +0032.0   +0000.1
```
Comment (anacso17) [inline pwrsupply/pstypes-setpoint-limits.txt:102]: wouldn't it be `si-quadrupole-q30-qfb-fam` and `si-quadrupole-q30-qfp-fam`?

Comment (xresende) [reply pwrsupply/pstypes-setpoint-limits.txt:102]: yes! thanks.

Comment (xresende) [reply pwrsupply/pstypes-setpoint-limits.txt:102]: fixed, @anacso17

**Review by anacso17 — APPROVED** [review]

File: pwrsupply/pstypes-setpoint-limits.txt (hunk 1 above)
Comment (anacso17) [reply pwrsupply/pstypes-setpoint-limits.txt:102]: missing a push, maybe?

**Review by murilobalves — APPROVED** [review]

**Pull request merged**